admin/queue-scheduler/src/main/java/io/enmasse/queue/scheduler/Main.java

```diff
@@ -23,11 +23,12 @@ public static void main(String [] args) {
         Vertx vertx = Vertx.vertx();
         String configHost = getEnvOrThrow("CONFIGURATION_SERVICE_HOST");
         int configPort = Integer.parseInt(getEnvOrThrow("CONFIGURATION_SERVICE_PORT"));
-        int listenPort = 55667;
+        int listenPort = Integer.parseInt(getEnvOrThrow("LISTEN_PORT"));
 
         QueueScheduler scheduler = new QueueScheduler(
                 connection -> Artemis.create(vertx, connection),
-                listenPort);
+                listenPort,
+                getEnvOrThrow("CERT_DIR"));
```

Comment:
Do we need to throw here? - this precludes running without TLS even though the rest of the code looks like it would work without it